glusterd2/commands/global/commands.go

```diff
@@ -0,0 +1,35 @@
+// Package globalcommands implements the commands to get and set cluster level options
+package globalcommands
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/servers/rest/route"
+)
+
+// Command is a holding struct used to implement the GlusterD Command interface
+type Command struct {
+}
+
+// Routes returns command routes. Required for the Command interface.
+func (c *Command) Routes() route.Routes {
+	return route.Routes{
+		route.Route{
+			Name:        "SetGlobalOptions",
+			Method:      "POST",
+			Pattern:     "/cluster/options",
+			Version:     1,
+			HandlerFunc: setGlobalOptionsHandler,
+		},
+		route.Route{
+			Name:        "GetGlobalOptions",
+			Method:      "GET",
+			Pattern:     "/cluster/options/{optname}",
```

Comment:
Return all options at once, CLI can filter if required. (optname can be removed)